src/reactor.cc

```diff
@@ -66,6 +71,11 @@ void Reactor::EnterNotify() {
     }
   }
 
+  // Test if any side products have been defined.
+  if (side_products.size() == 0){
+    bool hybrid = false;
```

Comment:
Not clear what this does, since hybrid won't escape the if-scope

Reply:
and isn't used

Reply:
Ahh this is just setting a state-variable, so you probably want it to not-redeclare here:

```C++
hybrid = false;
```